src/app/landing/LandingPage.jsx

```diff
@@ -65,26 +65,36 @@ class LandingPage extends Component {
               <source
                 className="OnboardingImage"
                 media="(min-aspect-ratio: 2/3)"
-                srcSet="img/landing_page/map_tablet.png"
+                srcSet="img/landing_page/step1.png"
```

Comment:
Looking at the video, I see the Food and Shelter markers are truncated. Has that always been the case?

Reply:
The alternative is to look like this:
![image](https://user-images.githubusercontent.com/20152824/82693372-6d567500-9c2f-11ea-8da1-87c47aeb3a34.png)

I will ask Di for what she prefers

Reply:
You mean there'd be white padding at the bottom? That's almost certainly worse than some truncation.
However, I believe the previous versions of the pictures somehow managed to avoid both, thanks to having the right ratios and "padding" in the 2 different versions...
Maybe the ratios are different this time?

Probably best to try prod/test for a second with whatever device size you're using, and see if it has the same problem. If it does, then maybe we just need to accept this.